aws_lambda_powertools/utilities/parser/models/apigw_websocket.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+from typing import Dict, List, Literal, Optional, Type, Union
+
+from pydantic import BaseModel
+from pydantic.networks import IPvAnyNetwork
+
+
+class APIGatewayWebSocketEventIdentity(BaseModel):
```

Comment:
In my tests I see the user_agent always come inside Indentity:

`'identity': {'userAgent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36', 'sourceIp': '179.164.64.249'}`

Can you please check if it's Optional or not?

Reply:
Hey, I'm not near a computer in the next couple.of days. I attached the aws docs at tje start, could you verify it?